README.md

```diff
@@ -45,6 +45,28 @@ npm run build
 
 End with an example of getting some data out of the system or using it for a little demo
 
+## Handling feature flags
+
+To hide functionality that shouldn't be visible on production environment we are using feature flags handled entirely on
+frontend side. This is done by flag in environment file named `featureVisible`.
+To hide some part of html code you should use directive `appIsFeatureEnabled` which works like `*ngIf` directive.
+For example:
+```angular2html
```

Comment:
Cool, I didn't know about `angular2html` code snippet type 😮

Reply:
Webstorm suggested this :D